### ISSUE TITLE
Save highscores for the entire world

### ISSUE THREAD

**Issue by Galarzaa90** [Galarzaa90/NabBot#44]
Right now, highscores are only saved for registered characters, so when a new character is registered, it takes a while for their highscores to appear.

Checking the highscores is a heavy operation, that is already done, storing all of them would take practically no resources, and highscores would be available for any player (in tracked worlds).

Aditionally, by storing all characters, highscores could be checked only once per server save, by storing the last day they were checked, reducing the number of calls made to tibia.com and reducing networks errors caused by the frequent number of calls.

A new database table for highscores would be created
**highscores**

 rank | category | world | name | vocation | value
--- | --- | --- | --- | --- | --- 
2 | shielding | Fidera | Delita Heirall | Elite Knight | 120

**highscores_times**

 world | last_scan | category
--- | ---  | ---
Fidera | 1501271274 | shielding

Progress:  
- [x] Create tables
- [x] Modify scan highscores task
- [x] Modify /whois to read from new table
